### PR TITLE
Platform x86-64-agema-agc7648

### PR DESCRIPTION
**Pull request by spitchai**
```
updated to the new ONIE platform string
```

@sonoble @jnealtowns

### CODE REVIEW

File: packages/platforms/agema/x86-64/x86-64-agema-agc7648a/onlp/PKG.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 variables:
-  platform: x86-64-agc7648-r0
+  platform: x86-64-agc7648a-r0
```
Comment (jnealtowns) [inline packages/platforms/agema/x86-64/x86-64-agema-agc7648a/onlp/PKG.yml]: This does not follow the ONIE platform specification. No separate vendor-platform tokens. Is this really their identifier?

Comment (spitchai) [inline packages/platforms/agema/x86-64/x86-64-agema-agc7648a/onlp/PKG.yml]: Yes. This is what i got. 

root@localhost:~#
root@localhost:~# onie-shell
EXT4-fs (sdb2): mounted filesystem with ordered data mode. Opts: (null)
/ # onie-sysinfo
x86_64-agc7648a-r0
/ #

it used to be x86_64-agc7648-r0 now  it is x86_64-agc7648a-r0

File: packages/platforms/agema/x86-64/x86-64-agema-agc7648a/platform-config/r0/src/lib/x86-64-agc7648a-r0.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ x86-64-agc7648-r0:
       --parity=0
       --stop=1
 
-    kernel: 
+    kernel:
```
Comment (jnealtowns) [inline packages/platforms/agema/x86-64/x86-64-agema-agc7648a/platform-config/r0/src/lib/x86-64-agc7648a-r0.yml]: @spitchai  The 3.18 kernel is going to be deprecated. Only the 3.16 and future 4.X kernels will be supported. 
Can you use the 3.16 kernel instead?

Comment (spitchai) [inline packages/platforms/agema/x86-64/x86-64-agema-agc7648a/platform-config/r0/src/lib/x86-64-agc7648a-r0.yml]: Sure.

**Comment by spitchai** [comment]
@sonoble @jnealtowns 
Agema folks want to pull the code. can we gets this merged.

**Comment by jnealtowns** [comment]
@spitchai These changes have been integrated. I'm afraid its still not clear whether the ONIE platform identifier is correct or not.

**Pull request closed without merging**